src/schema/create-table-builder.ts

```diff
@@ -162,7 +162,7 @@ export class CreateTableBuilder<TB extends string, C extends string = never>
    * ```
    */
   addPrimaryKeyConstraint(
-    constraintName: string,
+    constraintName?: string,
     columns: C[]
   ): CreateTableBuilder<TB, C> {
```

Comment:
This is not valid TypeScript.

You should split to multiple overloads as follow:

```ts
addPrimaryKeyConstraint(constraintName: string, columns: C[]): CreateTableBuilder<TB, C>
addPrimaryKeyConstraint(columns: C[]): CreateTableBuilder<TB, C>

addPrimaryKeyConstraint(...args: any[]): any {
  // implementation
}
```